### ISSUE TITLE
Make the Paypal script-src reference more explicit

### ISSUE THREAD

**Issue by muffinresearch** [mozilla/addons-server#3461]
We can limit the source of script for paypal by referencing the full script rather than white-listing the complete origin.

**Comment by muffinresearch** [comment]
QA notes: 

Ignoring time-outs contributions should still function and the paypal script should not get blocked by this change.

**Comment by ValentinaPC** [comment]
@muffinresearch : Contributions work.
I've checked on AMO-dev using FF48.0.2(Win 7x64)

There is a small issue, though, don't really think that is related to this change, but not sure.
Please see the screenshot and let me know if a separate issue is required. Thanks!
Screenshot: 
![japan](https://cloud.githubusercontent.com/assets/15685960/18432416/867e948e-78ea-11e6-9a61-44540b51e850.png)

This is happening when entering a very small value for contribution.

**Comment by muffinresearch** [comment]
> There is a small issue, though, don't really think that is related to this change, but not sure.
Please see the screenshot and let me know if a separate issue is required. Thanks!

@ValentinaPC That error message appears to come from paypal. It would be interesting to know if we can reproduce the same thing in production. If we can it's worth filing a bug , but if not it might be just something related to their sandbox.

**Comment by ValentinaPC** [comment]
I was able to reproduce this in AMO-prod, also, after a few attempts.
![2016-09-12_1425](https://cloud.githubusercontent.com/assets/15685960/18434281/e60c6958-78f4-11e6-8ea6-8820d4cf77b5.png)

But, I am not being able to find a pattern. :(

**Comment by ValentinaPC** [comment]
Verified as fixed on AMO-dev FF48.0.2(Win 7 x64) - Contributions work as expected.
Logged another issue for the multiple lang message (#3481)
